docs/backup_and_restore.rst

```diff
@@ -101,89 +88,257 @@ When you are ready to begin the backup, run
+migration using a v2-only backup. However, it is possible to migrate submissions,
+source accounts, and journalist accounts. To do so, follow the steps below:
+
+.. note:: The instructions below assume that you are using the same *Admin Workstation*
+  that was used to manage your old instance. If you are using a new *Admin
+  Workstation* you will need to copy the directory ``~amnesia/Persistent/securedrop``
+  from the old workstation to the new workstation (using a *Transfer Device*)
+  before proceeding.
+
+#. If you have not already done so, :ref:`back up the existing installation <backing_up>`. The instructions below assume that the backup has been created and renamed ``sd-backup-old.tar.gz``.
 
-Make sure to replace ``sd-backup-2017-07-22--01-06-25.tar.gz`` with the filename
-for your backup archive. The backup archives are located in
-``install_files/ansible-base``.
+#. Move the existing *Admin Workstation* SecureDrop code out of the way, by
+   opening a Terminal via **Applications > System Tools > Terminal** and
+   running the command:
 
-Once the restore is done, the *Application Server* will use the original
-*Source Interface* and *Journalist Interface* Onion URLs. You will need
-to update the corresponding files on the *Admin Workstation*:
+   .. code:: sh
 
-* ``app-source-ths``
-* ``app-journalist-aths``
-* ``app-ssh-aths``
+      mv ~/Persistent/securedrop ~/Persistent/sd.bak
 
-Once SSH access to the servers has been established (or if using SSH over
-local network), Onion URLs for the *Source Interface* and *Journalist Interfaces*
-can be fetched using the installer:
+#. Move the existing *Admin Workstation* SSH configuration out of the way via
+   the Terminal, using the commands:
+
+   .. code:: sh
+
+       mv ~/.ssh/config ~/.ssh/config.bak
+       mv ~/.ssh/known_hosts ~/.ssh/known_hosts.bak
+
+#. Reinstall SecureDrop  on the *Admin Workstation* using the following Terminal
+   commands:
+
+   .. code:: sh
+
+      cd ~/Persistent
+      git clone https://github.com/freedomofpress/securedrop
+
+#. Verify that the current release tag was signed with the release signing key:
+
+   .. code:: sh
+
+      cd ~/Persistent/securedrop/
+      git fetch --tags
+      git tag -v 1.7.1
+
+   The output should include the following two lines:
+
+   .. code:: sh
+
+      gpg:                using RSA key 22245C81E3BAEB4138B36061310F561200F4AD77
+      gpg: Good signature from "SecureDrop Release Signing Key"
+
+
+   .. important::
+      If you do not see the message above, signature verification has failed
+      and you should **not** proceed with the installation. If this happens,
+      please contact us at securedrop@freedom.press.
+
+
+   Verify that each character of the fingerprint matches what is on the
+   screen of your workstation. If it does, you can check out the new release:
+
+   .. code:: sh
+
+      git checkout 1.7.1
+
+   .. important::
+      If you see the warning ``refname '1.7.1' is ambiguous`` in the
```

Comment:
As above.

Reply:
Added this doc to `update_version.sh`, so all such should be resolved (thank you for this tip!)